Microsoft.Toolkit.Uwp.UI.Behaviors/VoiceCommands/VoiceCommandTrigger.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.Xaml.Interactivity;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Uwp.UI.Behaviors
+{
+    public class VoiceCommandTrigger : Trigger
+    {
+        private static readonly Dictionary<string, List<VoiceCommandTrigger>> _triggers = new Dictionary<string, List<VoiceCommandTrigger>>(StringComparer.InvariantCultureIgnoreCase);
+        private static ISpeechRecognizer _speechRecognizer;
+
+        public static ISpeechRecognizer SpeechRecognizer
+        {
+            get => _speechRecognizer;
+            set
+            {
+                if (value != _speechRecognizer)
+                {
+                    if (_speechRecognizer is object)
+                    {
+                        _speechRecognizer.Recognized -= SpeechRecognizer_Recognized;
+                    }
+                    _speechRecognizer = value;
+                    if (_speechRecognizer is object)
+                    {
+                        _speechRecognizer.Recognized += SpeechRecognizer_Recognized;
+                    }
+                }
+            }
+        }
+
+        private static void SpeechRecognizer_Recognized(ISpeechRecognizer sender, RecognizedEventArgs e)
+        {
+            Debug.WriteLine(e.Result.Text);
+            if (_triggers.TryGetValue(e.Result.Text, out var list))
+            {
+                foreach (var trigger in list)
+                {
+                    _ = trigger.Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () =>
+                    {
+                        Interaction.ExecuteActions(trigger.AssociatedObject, trigger.Actions, e);
+                    });
+                }
+            }
+        }
+
+        public string Text
```

Comment:
Would it make sense to turn this into a list for better maintainability / readability?

So instead of:
<Behaviors:VoiceCommandTrigger Text="Move available to First|Move available too First">

something like:
<Behaviors:VoiceCommandTrigger>
   <Behaviors:VoiceCommandTrigger.Commands>
<Command Text="Move available to First"/>
<Command Text="Move available too First"/>
 </Behaviors:VoiceCommandTrigger.Commands>
 </Behaviors:VoiceCommandTrigger>
 
I have to say that's way more verbose, which isn't great at all.. but might be easier to append a string/voice command to a list programmatically instead of re-formatting a string.

Reply:
I have thought about this too. I decided not to implement this. It would make localization much harder. With my | (pipe) separator you can easily add or remove alternatives for different languages. This will be very hard if you use a Commands collection.

Reply:
Should we still have the type be a list and use a custom converter to parse the text and split it?

https://timheuer.com/blog/implement-type-converter-uwp-winrt-windows-10-xaml

Would it also make sense to make the delimiter a property that can be overridden? Comma by default seems like a better choice?